project.clj

```diff
@@ -1,11 +1,11 @@
-(defproject prismatic/schema-generators "0.1.2-SNAPSHOT"
+(defproject prismatic/schema-generators "0.1.3-SNAPSHOT"
```

Comment:
This was correct as-is; 0.1.2 will be the next release.

Reply:
updated